Examples/demo-chat-app/demo-chat-app/MomentoDemoChatApp.swift

```diff
@@ -0,0 +1,36 @@
+import SwiftUI
+import momento
+
+class Momento: ObservableObject {
+    private let momentoApiKey: String = ""
```

Comment:
User will need to provide their Momento API key here in order to run the app. Can switch to using a token vending machine in the future when we implement `generateDisposableToken`

Reply:
I think we should add a note this in the README letting users know where to put the key.